reference/configuration/tidb-server/configuration-file.md

```diff
@@ -16,18 +16,29 @@ The TiDB configuration file supports more options than command-line parameters.
 - Default value: `true`
 - It is recommended to set it to `false` if you need to create a large number of tables.
 
-### `oom-action`
-
-- Specifies the operation when out-of-memory occurs in TiDB.
-- Default value: `log`
-- The valid options are `log` and `cancel`. `log` only prints the log without actual processing. `cancel` cancels the operation and outputs the log.
-
 ### `mem-quota-query`
 
 - The maximum memory available for a single SQL statement.
 - Default value: `34359738368`
 - Requests that require more memory than this value are handled based on the behavior defined by `oom-action`.
 
+### `oom-use-tmp-storage`
+
++ Set to `true` to enable use of temporary disk for some executors when `mem-quota-query` is exceeded.
++ Default value:  `true`
++ Valid options: `true` or `false`
```

Comment:
I think the `mem-quota-query` above should be referenced.

Reply:
Should the `Valid options` be kept?

Reply:
`mem-quota-query` can be referenced. I update the comment.
This is a Boolean variable, right? So "valid options" is kind of unnecessary unless you want to tell users when it is suitable to set it to `false` or `true`.

Reply:
But users do not know it's a Boolean variable.

Reply:
Actually, this doc is not that well-formatted. Different configuration items are introduced in different ways...
Generally, each configuration item has three parts: 
* Explanation
* Default value
* Note (optional)

So my suggestion is to cut the "valid option" out. But still, you can keep it since this doc is not well-formatted.